homeassistant/components/fritz/const.py

```diff
@@ -0,0 +1,38 @@
+"""Constants for the FRITZ!Box Tools integration."""
+
+DOMAIN = "fritzbox_tools"
```

Comment:
you need ti change the DOMAIN to fritz :D

Reply:
It works once this line and the one below are changed.

Reply:
Dunno how it slipped it. Thx!